eskipfile/watch_test.go

```diff
@@ -45,7 +45,7 @@ func deleteFile(t *testing.T) {
 }
 
 func createFileWith(content string, t *testing.T) {
-	tmpfile, err := ioutil.TempFile("", "watch_test-*")
+	tmpfile, err := os.Create(tmpTestWatchFile)
```

Comment:
minor: maybe derive the name like `testWatchFile + ".tmp"` to avoid new constant